packages/openactive-broker-microservice/app.js

```diff
@@ -987,6 +983,28 @@ function detectOpportunityType(opportunity) {
   }
 }
 
+/**
+ * @param {{[k: string]: any}} opportunity
+ * @returns {string[]} An opportunity can support multiple Booking Flows as these are in the Offers. An Opportunity
+ *   can have an Offer requiring approval and one that does not.
+ */
+function detectOpportunityBookingFlows(opportunity) {
```

Comment:
An Opportunity can be put into both booking flow buckets if it has Offers which support both